### PR TITLE
Ltpa key absolute and relative path tests

### LINKED ISSUE #26345: Test Debt: LTPA keysFileName is not tested with an absolute path
Currently no FAT tests are using an absolute path for the ltpa `keysFileName` config attribute. All existing tests seem to either be using the default value or a relative path.

This was discovered by a recent regression in 23.0.0.9: https://github.com/OpenLiberty/open-liberty/issues/26341

Example of absolute path
`<ltpa keysFileName="/opt/path/to/ltpa.keys" />`

Example of relative path
`<ltpa keysFileName="${server.config.dir}/ltpa.keys" />`

**Required Action**
1. Add or modify a FAT testcase to use an absolute path for the ltpa `keysFileName` config attribute value.

### PR DESCRIPTION
**Pull request by mshah0722**
LTPA Key Rotation Fat Tests

What we're testing: `<ltpa keysFileName="${server.config.dir}/resources/security/ltpa.keys" monitorInterval = "5" monitorDirectory="true" />`

Adding 2 more tests to: `LTPAKeyRotationTests.java`
1. `testDifferentDirectoriesForPrimaryKeys`
2. `testDifferentDirectoriesForValidationKeys`

### CODE REVIEW

**Comment by mshah0722** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.token.ltpa_fat

**Comment by mshah0722** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.token.ltpa_fat

File: dev/com.ibm.ws.security.token.ltpa_fat/fat/src/com/ibm/ws/security/token/ltpa/fat/LTPAKeyRotationTests.java [hunk 1]
```diff
@@ -1602,7 +1825,7 @@ public static void updateConfigDynamically(LibertyServer server, ServerConfigura
         String logLine = server.waitForStringInLogUsingMark("CWWKG001[7-8]I");
 
         // Wait for feature update to be completed or LTPA configuration to get ready
-        Thread.sleep(200);
```
Comment (Zech-Hein) [inline dev/com.ibm.ws.security.token.ltpa_fat/fat/src/com/ibm/ws/security/token/ltpa/fat/LTPAKeyRotationTests.java:1605]: we should try to remove this sleep if possible, it should wait long enough once the right message is detected, like LTPA config ready

**Review by Zech-Hein — APPROVED** [review]
LGTM

**Pull request merged**